src/main/java/se/sundsvall/casedata/service/AppealService.java

```diff
@@ -0,0 +1,55 @@
+package se.sundsvall.casedata.service;
+
+import io.github.resilience4j.retry.annotation.Retry;
+import org.springframework.stereotype.Service;
+import org.zalando.problem.Problem;
+import org.zalando.problem.Status;
+import org.zalando.problem.ThrowableProblem;
+import se.sundsvall.casedata.api.model.AppealDTO;
+import se.sundsvall.casedata.api.model.PatchAppealDTO;
+import se.sundsvall.casedata.integration.db.AppealRepository;
+
+import static se.sundsvall.casedata.service.util.mappers.EntityMapper.toAppealDto;
+import static se.sundsvall.casedata.service.util.mappers.PatchMapper.patchAppeal;
+import static se.sundsvall.casedata.service.util.mappers.PutMapper.putAppeal;
+
+@Service
+public class AppealService {
+
+	private static final ThrowableProblem APPEAL_NOT_FOUND_PROBLEM = Problem.valueOf(Status.NOT_FOUND, "Appeal not found");
```

Comment:
Why should we have an exception allocated in memory?
Why not just create it on the fly if necessary. 

The "create problem"-operation is not expensive and is not a part of the normal execution flow (it's an exception).